internal/backend/workflows/common.go

```diff
@@ -137,6 +139,7 @@ func (wf *Workflow) StartSessions(wctx workflow.Context, event sdktypes.Event, s
 }
 
 func (wf *Workflow) CreateEventRecord(ctx context.Context, eventID sdktypes.EventID, state sdktypes.EventState) {
+	ctx = cctx.WithRequestOrinator(ctx, cctx.Workflow)
```

Comment:
why do you need it here ?
don't you need to define it only once for the workflow?